install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -24,13 +24,23 @@ data:
     # Deprecated: mixer is using EDS
     {{- if .Values.mixer.enabled }}
     {{- if .Values.global.controlPlaneSecurityEnabled }}
+    {{- if .Values.global.proxy.discoveryDomain }}
+    mixerCheckServer: istio-policy.{{ .Release.Namespace }}.svc.{{ .Values.global.proxy.discoveryDomain }}:15004
+    mixerReportServer: istio-telemetry.{{ .Release.Namespace }}.svc.{{ .Values.global.proxy.discoveryDomain }}:15004
+    {{- else }}
     mixerCheckServer: istio-policy.{{ .Release.Namespace }}.svc.cluster.local:15004
```

Comment:
is mixerCheckServer and mixerReportServer not deprecated?  My understanding was this was all handled via EDS currently.  If so, we should probably remove it as to reduce confusion.